src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx

```diff
@@ -2,19 +2,35 @@ import React from "react";
 import PropTypes from "prop-types";
```

Comment:
File comment

Reply:
Updated in 187bf056d0a6139ae0a9a8ff06933a64eaf374c6